src/domains/issue/controller/issue.controller.ts

```diff
@@ -26,6 +30,15 @@ issueRouter.post('/dev/:userId/project/:projectId', validateRequest(UserProjectP
   return res.status(HttpStatus.OK).json(issues);
 });
 
+issueRouter.post('/pm/:userId/project/:projectId', validateRequest(UserProjectParamsDTO, 'params'), validateRequest(PMOptionalIssueFiltersDTO, 'body'), async (req: Request<UserProjectParamsDTO, any, PMOptionalIssueFiltersDTO>, res: Response) => {
```

Comment:
Same as before, filtering should be provided by query params and request should be a HTTP GET. Just a comment. POST is okay.

Reply:
Yes, I had already finished this endpoint when you made last recommendation. For future endpoints, we will user the library.